static/app/components/modals/widgetViewerModal/widgetViewerTableCell.tsx

```diff
@@ -37,7 +39,32 @@ type Props = {
   tableData?: TableDataWithTitle;
 };
 
-export const renderGridHeaderCell =
+export const renderIssueGridHeaderCell =
+  ({location, widget, tableData}: Props) =>
+  (column: TableColumn<keyof TableDataRow>, _columnIndex: number): React.ReactNode => {
+    const tableMeta = tableData?.meta;
+    const align = fieldAlignment(column.name, column.type, tableMeta);
+    const sortField = getSortField(String(column.key));
+
+    return (
+      <SortLink
+        align={align}
+        title={<StyledTooltip title={column.name}>{column.name}</StyledTooltip>}
+        direction={widget.queries[0].orderby === sortField ? 'desc' : undefined}
+        canSort
```

Comment:
Can we use `sortField` here to set `canSort={!!sortField}` so we don't have to do the ternary in `generateSortLink`?

From what I understand if you can't sort, then it won't render with a clickable header

Reply:
Yup, updated canSort and generateSortLink!